### PR TITLE
Integrate WeareTalisman wallet into the dApp

### PR DESCRIPTION
**Pull request by AtelyPham**
## Summary of changes
- I have integrated the WeareTalisman wallet into the dApp.
- Fix the bridge deposit error (thanks to @AhmedKorim's help). Really appreciate 🙏 .

### Reference issue to close (if applicable)
- Closes https://github.com/webb-tools/webb-dapp/issues/479

-----
### Code Checklist 

- [x] Tested: Videos demo below
- [x] Documented: [Steps to add a new wallet to Dapp](https://www.notion.so/hicommonwealth/Steps-to-add-a-new-wallet-to-Dapp-5c35b7d8c48d41fa8186c9ca3090b43e).

### Demo

- Wallet switching

https://user-images.githubusercontent.com/60747384/185695180-6cbe5cc1-1f68-4ea2-8318-c83f0c4eb77d.mp4

- The bridge deposit and withdraw

https://user-images.githubusercontent.com/60747384/185695288-7bafa6da-3301-4ff8-8d45-2c20170b4a1b.mp4

- Wrap/unwrap

https://user-images.githubusercontent.com/60747384/185695390-ee846289-811a-4a26-96a4-2554dfe4ee3a.mp4

### CODE REVIEW

**Comment by dutterbutter** [comment]
@AtelyPham what is remaining for this PR to be ready for review.

**Comment by AtelyPham** [comment]
I wasn't able to test my environment because Ahmed’s PR added new currencies on the dApp.

He also added a new script in `webb.js` to setup the local environment.

I will test it again and then mark this PR ready for review

File: packages/api-providers/src/ext-providers/polkadot/polkadot-provider.ts [hunk 1]
```diff
@@ -250,17 +256,30 @@ export class PolkadotProvider extends EventBus<ExtensionProviderEvents> {
...
+        case WalletId.Polkadot: {
+          throw WebbError.from(WebbErrorCodes.PolkaDotExtensionNotInstalled);
+        }
+
+        case WalletId.Talisman: {
+          throw WebbError.from(WebbErrorCodes.TalismanExtensionNotInstalled);
+        }
+
+        default: {
+          throw new Error('Unknown wallet');
```
Comment (AhmedKorim) [inline packages/api-providers/src/ext-providers/polkadot/polkadot-provider.ts:274]: Can we have a typed error here   **EX** `throw WebbError.from(WebbErrorCodes.UnkownWallet)`

File: packages/react-environment/src/error/interactive-errors/extension-not-installed.tsx [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
+      return walletsConfig[WalletId.Polkadot];
+
+    case WebbErrorCodes.MetaMaskExtensionNotInstalled:
+      return walletsConfig[WalletId.MetaMask];
+
+    case WebbErrorCodes.TalismanExtensionNotInstalled:
+      return walletsConfig[WalletId.Talisman];
+
+    default:
+      throw new Error('Unknown wallet');
```
Comment (AhmedKorim) [inline packages/react-environment/src/error/interactive-errors/extension-not-installed.tsx:21]: Same here use typed WebbError

**Review by AhmedKorim — CHANGES_REQUESTED** [review]

**Review by AhmedKorim — APPROVED** [review]
LGTM!

**Pull request merged**